packages/textfield/stories/textarea.stories.ts

```diff
@@ -181,7 +181,11 @@ export const with5Rows = (): TemplateResult => html`
     <sp-textfield
         multiline
         id="predefinedRows"
-        value="Line 1\nLine 2\nLine 3\nLine 4\nLine 5"
+        value="Line 1
+Line 2
+Line 3
+Line 4
+Line 5"
```

Comment:
I don't know why some files were submitted together with my PR, they seem to be changes after linting...